src/services/exchange.js

```diff
@@ -149,7 +149,7 @@ export default class ExchangeService {
   }
 
   getShapeshiftAuthUrl() {
-    const { sub: regId } = extractJwtPayload(this.tokens.accessToken) || {};
+    const { sub: regId } = extractJwtPayload(this.tokens.accessToken);
```

Comment:
What if we fail to parse the JSON object inside `extractJwtPayload()` return?

Reply:
It will throw an exception? The signature of `extractJwtPayload` states that it will always return an `Object`.

Reply:
Yes, as the signature states we always get an object back, then we need to sure that we return that object in all the cases.
TLDR: just wrap the return with try .. catch

Reply:
Oh, you mean updating `extractJwtPayload`? I can do that :+1: